### DISCUSSION TITLE
Merge two confirmation table TPLs

### DISCUSSION THREAD

**Discussion by Hlavtox** [PrestaShop/hummingbird#820] [Ideas]
Use order-detail-no-return.tpl instead of order-confirmation-table.tpl, why have two tables.

**Comment by kpodemski** [comment]
@Zoefinders, what does it mean that the issue is in the "Archive"? it is not needed? it shouldn't be done?

**Comment by Zoefinders** [comment]
In "Arichive" these are issues that were written by @Hlavtox . 
I needed to make a clean up so that _theme & design_ team could get finish to work. 
@Hlavtox  told me to only take care of the elements of ticket 409. I archived the rest so I wouldn't delete it if needed. 
Maybe I can rename the folder or delete the tickets if your think is better? 
 @kpodemski @Hlavtox  What do you think ?
